### PR TITLE
[IA-4950] Update R version on Azure VMs to 4.4.0

### PR DESCRIPTION
**Pull request by lucymcnatt**
__Jira ticket__: https://broadworkbench.atlassian.net/browse/IA-4950

### What

- Manually installs an up-to-date R version in the Azure init script

### Why

- There's a security bug in lower versions

### What to test

- 

### Who tested and where

- [ ] This change is covered by automated tests 
  - _NB: Rerun automation tests on this PR by commenting `jenkins retest` or `jenkins multi-test`._
- [ ] I validated this change 
- [ ] Primary reviewer validated this change 
- [ ] I validated this change in the dev environment

### CODE REVIEW

File: http/src/main/resources/init-resources/azure_vm_init_script.sh [hunk 1]
```diff
@@ -169,6 +169,18 @@ while sudo fuser /var/lib/dpkg/lock-frontend > /dev/null 2>&1
...
 
+# Install updated R version
+
+## Switch to use Debian unstable - neccessary to install R 4.4.0 since it was just released 4/24/24
+echo "deb http://http.debian.net/debian sid main" > /etc/apt/sources.list.d/debian-unstable.list \
+    && echo 'APT::Default-Release "testing";' > /etc/apt/apt.conf.d/default \
+    && echo 'APT::Install-Recommends "false";' > /etc/apt/apt.conf.d/90local-no-recommends
+
+# Install R
+apt-get update && apt-get install -y -t unstable --no-install-recommends \
```
Comment (lucymcnatt) [inline http/src/main/resources/init-resources/azure_vm_init_script.sh]: R version on my BEE was still 4.3, may need to remove old R version first?

File: http/src/main/resources/init-resources/azure_vm_init_script.sh [hunk 2]
```diff
@@ -169,6 +169,15 @@ while sudo fuser /var/lib/dpkg/lock-frontend > /dev/null 2>&1
...
   done
 
+# Install updated R version
+echo "Installing R 4.4.0..."
+# Add the CRAN repository to the sources list
+echo "deb https://cloud.r-project.org/bin/linux/ubuntu focal-cran40/" | sudo tee /etc/apt/sources.list -a
+# Update package list
+sudo apt-get update
+# Install new R version
+sudo apt install --no-install-recommends -y r-base=4.4.0-1.2004.0
```
Comment (LizBaldo) [inline http/src/main/resources/init-resources/azure_vm_init_script.sh]: Nit, but could you store `4.4.0-1.2004.0` in an `R_Version` variable to make it easier to change, and also echo?
something like
R_version = `4.4.0-1.2004.0`

echo "Installing R ${R_version}"
...
sudo apt install --no-install-recommends -y r-base=${R_version}

**Review by LizBaldo — APPROVED** [review]
Thanks for doing this! I have a small suggestion to parametrize the version, but definitely not a blocker for merging

**Review by erdalcosgun — APPROVED** [review]

File: http/src/main/resources/init-resources/azure_vm_init_script.sh (hunk 2 above)
Comment (lucymcnatt) [reply http/src/main/resources/init-resources/azure_vm_init_script.sh]: oh yeah that makes sense!

**Pull request merged**